### PR TITLE
fix(dropdown): observe select attribute changes

### PR DESCRIPTION
**Pull request by lubber-de**
## Description
The select observer in dropdown did not detect attribute changes  of `option` or `optgroup` changes if the dropdown was build out of a `select` tag

## Testcase
- Open the dropdown and see an option group "Swedish Cars" and the "Rabbit" entry being available
- Close the dropdown
- Click on the "Change select tag" button
- Open the Dropdown again
 
## Broken
- Nothing has changed
https://jsfiddle.net/lubber/hjdv6bnL/8/

## Fixed
- "Swedish Cars" changed into "I was changed"
- "Rabbit" is disabled
https://jsfiddle.net/lubber/hjdv6bnL/9/

## Screenshot
|Broken|Fixed|
|-|-|
|selectobserverbroken|selectobserverfixed|

## Closes
#2033

### CODE REVIEW

File: src/definitions/modules/dropdown.js [hunk 1]
```diff
@@ -200,6 +200,7 @@ $.fn.dropdown = function(parameters) {
           select: function() {
             if(module.has.input() && selectObserver) {
               selectObserver.observe($module[0], {
+                attributes: true,
                 childList : true,
                 subtree   : true
               });
```
Comment (exoego) [inline src/definitions/modules/dropdown.js:206]: Note) See https://developer.mozilla.org/en-US/docs/Web/API/MutationObserverInit for options.

**Review by exoego — DISMISSED** [review]
LGTM

**Comment by lubber-de** [comment]
@exoego I made sure the observer methods are used centrally, please review again

**Review by prudho — APPROVED** [review]
LGTM

**Pull request merged**